presto-hive/src/test/sql/create-test-hive12.sql

```diff
@@ -29,6 +30,7 @@ SELECT
 , CASE n % 3 WHEN 0 THEN false WHEN 1 THEN true ELSE NULL END
 , CASE WHEN n % 17 = 0 THEN NULL ELSE '2011-05-06 07:08:09.1234567' END
 , CASE WHEN n % 23 = 0 THEN NULL ELSE CAST('test binary' AS BINARY) END
+, CASE n % 39 WHEN 0 THEN NULL WHEN 1 THEN '' ELSE 'test varchar' END
```

Comment:
value 39 is already being used for this column in create-test-hive13.sql. 37 is used by some dates